trade-finance-manager-api/src/v1/rest-mappings/mappings/facilities/mapFacilitySnapshot.js

```diff
@@ -55,7 +55,7 @@ const mapFacilitySnapshot = (facility, dealSnapshot) => {
 
   clonedSnapshot.facilityProduct = mapFacilityProduct(type);
 
-  clonedSnapshot.type = mapFacilityType(clonedSnapshot);
+  clonedSnapshot.type = mapFacilityType(type, facilitySnapshot);
```

Comment:
I'm aiming to reduce the need to use this 'clonedSnapshot' so that we can ultimately remove it